tar/generate_tar.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# locate config file
+CONFIG_FILE="config-example.yaml"
```

Comment:
I would make `CONFIG_FILE` a parameter with "config-example.yaml" as the default value, so users can pass in a custom config if they like.

Reply:
moved CONFIG_FILE declaration to make, so the variable can be passed in through make command